deployments/qat_plugin/base/intel-qat-plugin.yaml

```diff
@@ -18,28 +18,6 @@ spec:
         image: intel/intel-qat-plugin:devel
         securityContext:
           readOnlyRootFilesystem: true
-        env:
-        - name: DPDK_DRIVER
-          valueFrom:
-            configMapKeyRef:
-              name: intel-qat-plugin-config
-              key: DPDK_DRIVER
-        - name: KERNEL_VF_DRIVERS
-          valueFrom:
-            configMapKeyRef:
-              name: intel-qat-plugin-config
-              key: KERNEL_VF_DRIVERS
-        - name: MAX_NUM_DEVICES
-          valueFrom:
-            configMapKeyRef:
-              name: intel-qat-plugin-config
-              key: MAX_NUM_DEVICES
-        - name: VERBOSITY
-          valueFrom:
-            configMapKeyRef:
-              name: intel-qat-plugin-config
-              key: VERBOSITY
-
```

Comment:
Is it not needed anymore?

Reply:
Yes, @mythi suggested to remove it.

Reply:
we can drop the `ConfigMap` too. Maybe I could do this and then you can rebase?

Reply:
yes, please